Components/Image-Component/image.css

```diff
@@ -0,0 +1,25 @@
+.responsive-img {
+    width: 100%;
+    height: auto;
+}
```

Comment:
Hey you can add a property of object fit, so that it helps to resize img to fit its container. Code looks good to me. A preview would help more.

Reply:
Yes, will do that thanks!